src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Resolvers/DynamicObjectResolver.cs

```diff
@@ -1786,7 +1786,19 @@ EmittableMember CreateEmittableMember(MemberInfo m)
                         }
 
                         var getMethod = property.GetGetMethod(true);
+                        if (getMethod == null)
+                        {
+                            //check if property from base class has private GetMethod
+                            getMethod = property.DeclaringType.GetProperty(property.Name, BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.Static).GetGetMethod(true);
```

Comment:
I don't think we want `Static` properties. A serializer has no business handling static data.